### PR TITLE
Add Contribution Guide

### PR DESCRIPTION
**Pull request by codie3611**
This PR adds a contribution guide to the repo and the docs.﻿

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -41,6 +41,7 @@ nav:
       - Overview: diffcrash/diffcrash.md
       - Command Line Tool: diffcrash/run.md
       - Code: diffcrash/diffcrash_run.md
+  - Contribution Guide: contributing.md
```
Comment (codie3611) [inline mkdocs.yml:44]: Add the contribution guide also to the docs

File: docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1 @@
+--8<-- "./CONTRIBUTING.md"
```
Comment (codie3611) [inline docs/contributing.md:1]: This inlines the `CONTRIBUTION.md` file from the repo root here

**Review by ballaln — COMMENTED** [review]
Looks good :)

**Comment by codie3611** [comment]
> Looks good :)

You can btw go on **Review** and approve there for future PRs

**Pull request merged**